tests/integration/test_events.py

```diff
@@ -57,8 +59,11 @@ def assert_valid_event(self, event):
         for field in expected_fields:
             assert field in event
 
+    def create_rule_name(self):
+        return "rule-{}".format(short_uid())
```

Comment:
```suggestion
        return f"rule-{short_uid()}"
```

Generally please use f-strings for generic string formatting. The only exception is (debug) logging where the logger's formatting system should be used in performance critical areas to unnecessary formatting.

That being said, we're generally trying to avoid extracting "util functions" for these trivial actions. They don't provide a significant benefit and increase complexity, especially in terms of cognitive load when reading code. 

You can think of this as trying to make the test as "standalone" as possible.